src/main/java/com/foundation/search/model/SearchResult.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (c) 2018 Jalasoft.
+ * 2643 Av Melchor Perez de Olguin, Colquiri Sud, Cochabamba, Bolivia.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of
+ * Jalasoft, ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ *  with Jalasoft.
+ */
+
+package com.foundation.search.model;
+
+public class SearchResult {
+    public String pathResult;
```

Comment:
Please update the access for the properties to "private" or at least "protected", and refactor the get methods accordingly